test/sql/storage/shutdown_create_index.test

```diff
@@ -2,7 +2,6 @@
 # description: CREATE INDEX statement after shutdown
 # group: [storage]
 
-# load the DB from disk
```

Comment:
Could we add a test that interleaves `NULL` values in the index in order to create higher row ids to stress test the handling of differently sized row ids?